tasks/config/project/config.js

```diff
@@ -128,18 +128,14 @@ module.exports = {
     }
 
     // takes component object and creates file glob matching selected components
-    config.globs.components = (typeof config.components == 'object')
-      ? (config.components.length > 1)
-        ? '{' + config.components.join(',') + '}'
-        : config.components[0]
+    config.globs.components = (Array.isArray(config.components) && config.components.length >= 1)
+      ? '{' + config.components.join(',') + '}'
```

Comment:
Note) One idea came up on my mind when working on this.
If we exclude the names, defined in `individuals`, from `components` here, I think users can define only `individuals` and cn omit `components`.
This makes it easier to use `individuals` when optimizing bundling.

Currently, one must define `components` (see below doc), but I think this is not great when new component is added.

> https://fomantic-ui.com/introduction/build-tools.html#excluding-components-from-main-cssjs-file
> Note: After defining the individuals in semantic.json, you need to remove them from the main components list. This also means that you can't leave the components list empty anymore (which defaults to building all components).

If this is OK, I will open a separate PR for this enhancement.

Reply:
Yes. That would be more friendly. If we've `individuals`, then everything except from the `individuals` should be bundled without needing to describe with `components`.

Reply:
Remember, the `components` settings defines all the wanted components inside the semantic.json. If we automatically create that list by substracting the `individuals` list form all available components, you are going to force people to always have every other component inside the semantic.css/.js.

### Example
- one wants to only have dimmer, modal, button, site and reset as `components´
- one also wants to have toast as individual

#### Current logic
- semantic.js/.css will contain the components dimmer, modal, button, side, reset
- toast.js/.css is generated separately

#### Your proposed logic
- semantic.js/.css contains **everything but** toast.js/.css
- toast.js/.css is generated separately

So most probably such change is only save/backward compatible, if no `components` value is given.

Reply:
That's right. To having other components in the semantic.css/js, the user would need to add all components into `individuals` except those he want to bundle. But it'll also end up with the unnecessary components get generated by build process.

Reply:
Oh, I do not mean removing `components` completely.
In the below matrix, No. 4 is my proposal.

\No. |components | individuals | Current with this bugfix | Proposal
---|---|---|---|--- 
1|`dimmer`, `button` | undefined/empty | `semantics.js/.css`: only `dimmer` and `button` | same as left
2|`dimmer`, `button` | `toast`| `semantics.js/.css`: only `dimmer` and `button`<br>`toast.js/.css` is generated separately | same as left
3|undefined/empty | undefined/empty | `semantics.js/.css`: all | same as left
4|undefined/empty | `toast`| `semantics.js/.css`: all including `toast` 😞<br>`toast.js/.css` is generated separately | `semantics.js/.css`: all minus `toast` 😄 <br>`toast.js/.css` is generated separately

Reply:
Alright, that's what i also meant by 
> So most probably such change is only save/backward compatible, if no components value is given.

I'll merge this PR here now, so, you can go ahead and prepare another one for the proposed solution above ;)